SmartDeviceLink_Example/Classes/ProxyManager.m

```diff
@@ -273,6 +275,31 @@ + (SDLAddCommand *)sdlex_vehicleDataCommandWithManager:(SDLManager *)manager {
     return getVehicleDataCommand;
 }
 
++ (SDLAddCommand *)sdlex_dialNumberCommandWithManager:(SDLManager *)manager {
+    NSString *menuName = @"Dial Number";
+    SDLMenuParams *commandMenuParams = [[SDLMenuParams alloc] initWithMenuName:menuName];
+
+    SDLAddCommand *dialNumberCommand = [[SDLAddCommand alloc] init];
+    dialNumberCommand.vrCommands = [NSMutableArray arrayWithObject:menuName];
+    dialNumberCommand.menuParams = commandMenuParams;
+    dialNumberCommand.cmdID = @3;
+    dialNumberCommand.handler = ^(SDLOnCommand * _Nonnull command) {
+        SDLLogD(@"Checking phone call capability");
+        [manager.systemCapabilityManager updateCapabilityType:SDLSystemCapabilityTypePhoneCall completionHandler:^(NSError * _Nullable error) {
+            if (error == nil) {
+                SDLLogD(@"Dialing number");
+                [self.class sdlex_sendDialNumberWithManager:manager];
```

Comment:
This also needs to check the permission manager. Most apps don't have permission for this, probably including this app.

Reply:
Fixed